src/util/support.js

```diff
@@ -1,5 +1,36 @@
+import isNativeRegex from 'regex-native-function'; // checks if a function is native using regex
```

Comment:
Not really, I was just exploring built-in ways to do this. Just trying to keep deps to a minimum, if possible.